src/fsharp/service/SemanticClassification.fs

```diff
@@ -24,30 +23,46 @@ type SemanticClassificationType =
     | ReferenceType
     | ValueType
     | UnionCase
+    | UnionCaseField
     | Function
     | Property
     | MutableVar
     | Module
+    | NameSpace
     | Printf
     | ComputationExpression
     | IntrinsicFunction
     | Enumeration
     | Interface
     | TypeArgument
     | Operator
-    | Disposable
+    | DisposableType
+    | DisposableValue
+    | Method
+    | ExtensionMethod
+    | Constructor
```

Comment:
I'll want to break this into at least two constructor types, `ClassConstructor` and `StructConstructor`

`DelegateConstructor` is also a possibility, and also I may want to consider what to do about `FakeInterfaceCtor` and if that should be special or not